RELEASENOTES.md

```diff
@@ -2,6 +2,9 @@
 Nett:
 + Add: Callback for processing non map-able table rows [#78](https://github.com/paiden/Nett/issues/78)
 
+AspNet:
```

Comment:
This should go into  a new 0.14.0 section

Reply:
My bad, I hadn't noticed.
It should be fixed now.